test_scripts/API/VehicleData/GetVehicleData/005_GetVD_fuelRange_params_missed.lua

```diff
@@ -0,0 +1,61 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies
+-- 2) HMI sends boundary values of parameters in response
```

Comment:
@GetmanetsIrina Please correct description according to script

Reply:
Done

Reply:
@GetmanetsIrina There is no new commits in this pull request

Reply:
@dboltovskyi, please check 0d91732b742b57867fa93347d9a12b3d67ccfc03 .